This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.

It was created by configure, which was
generated by GNU Autoconf 2.60.  Invocation command line was

  $ ./configure CPPFLAGS=-I/Users/kokim/anaconda3/envs/py37/include LDFLAGS=-L/Users/kokim/anaconda3/envs/py37/lib CFLAGS=-I/Users/kokim/anaconda3/envs/py37/include --prefix=/Users/kokim/anaconda3/envs/py37

## --------- ##
## Platform. ##
## --------- ##

hostname = kokimui-MacBookPro.local
uname -m = x86_64
uname -r = 18.5.0
uname -s = Darwin
uname -v = Darwin Kernel Version 18.5.0: Mon Mar 11 20:40:32 PDT 2019; root:xnu-4903.251.3~3/RELEASE_X86_64

/usr/bin/uname -p = i386
/bin/uname -X     = unknown

/bin/arch              = unknown
/usr/bin/arch -k       = unknown
/usr/convex/getsysinfo = unknown
/usr/bin/hostinfo      = Mach kernel version:
	 Darwin Kernel Version 18.5.0: Mon Mar 11 20:40:32 PDT 2019; root:xnu-4903.251.3~3/RELEASE_X86_64
Kernel configured for up to 8 processors.
4 processors are physically available.
8 processors are logically available.
Processor type: x86_64h (Intel x86-64h Haswell)
Processors active: 0 1 2 3 4 5 6 7
Primary memory available: 16.00 gigabytes
Default processor set: 419 tasks, 1760 threads, 8 processors
Load average: 1.32, Mach factor: 6.67
/bin/machine           = unknown
/usr/bin/oslevel       = unknown
/bin/universe          = unknown

PATH: /Users/kokim/anaconda3/envs/py37/bin
PATH: /opt/local/bin
PATH: /opt/local/sbin
PATH: /Users/kokim/anaconda3/condabin
PATH: /usr/local/bin
PATH: /usr/bin
PATH: /bin
PATH: /usr/sbin
PATH: /sbin
PATH: /Applications/VMware Fusion.app/Contents/Public


## ----------- ##
## Core tests. ##
## ----------- ##

configure:1720: checking for gcc
configure:1747: result: gcc
configure:1985: checking for C compiler version
configure:1992: gcc --version >&5
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:1995: $? = 0
configure:2002: gcc -v >&5
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.5.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
configure:2005: $? = 0
configure:2012: gcc -V >&5
clang: error: unsupported option '-V -Wno-atomic-implicit-seq-cst'
clang: error: no input files
configure:2015: $? = 1
configure:2038: checking for C compiler default output file name
configure:2065: gcc -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c  >&5
configure:2068: $? = 0
configure:2114: result: a.out
configure:2119: checking whether the C compiler works
configure:2129: ./a.out
configure:2132: $? = 0
configure:2149: result: yes
configure:2156: checking whether we are cross compiling
configure:2158: result: no
configure:2161: checking for suffix of executables
configure:2168: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c  >&5
configure:2171: $? = 0
configure:2195: result: 
configure:2201: checking for suffix of object files
configure:2227: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:2230: $? = 0
configure:2253: result: o
configure:2257: checking whether we are using the GNU C compiler
configure:2286: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:2292: $? = 0
configure:2299: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:2302: $? = 0
configure:2309: test -s conftest.o
configure:2312: $? = 0
configure:2326: result: yes
configure:2331: checking whether gcc accepts -g
configure:2361: gcc -c -g -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:2367: $? = 0
configure:2374: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:2377: $? = 0
configure:2384: test -s conftest.o
configure:2387: $? = 0
configure:2517: result: yes
configure:2534: checking for gcc option to accept ISO C89
configure:2608: gcc  -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:2614: $? = 0
configure:2621: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:2624: $? = 0
configure:2631: test -s conftest.o
configure:2634: $? = 0
configure:2654: result: none needed
configure:2716: checking for a BSD-compatible install
configure:2772: result: /usr/bin/install -c
configure:2785: checking for ar
configure:2801: found /usr/bin/ar
configure:2813: result: ar
configure:2828: checking how to run the C preprocessor
configure:2868: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
configure:2874: $? = 0
configure:2912: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
conftest.c:8:10: fatal error: 'ac_nonexistent.h' file not found
#include <ac_nonexistent.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
configure:2918: $? = 1
configure: failed program was:
| /* confdefs.h.  */
| #define PACKAGE_NAME ""
| #define PACKAGE_TARNAME ""
| #define PACKAGE_VERSION ""
| #define PACKAGE_STRING ""
| #define PACKAGE_BUGREPORT ""
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:2958: result: gcc -E
configure:2987: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
configure:2993: $? = 0
configure:3031: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
conftest.c:8:10: fatal error: 'ac_nonexistent.h' file not found
#include <ac_nonexistent.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
configure:3037: $? = 1
configure: failed program was:
| /* confdefs.h.  */
| #define PACKAGE_NAME ""
| #define PACKAGE_TARNAME ""
| #define PACKAGE_VERSION ""
| #define PACKAGE_STRING ""
| #define PACKAGE_BUGREPORT ""
| /* end confdefs.h.  */
| #include <ac_nonexistent.h>
configure:3082: checking for grep that handles long lines and -e
configure:3156: result: /usr/bin/grep
configure:3161: checking for egrep
configure:3239: result: /usr/bin/grep -E
configure:3244: checking for ANSI C header files
configure:3274: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3280: $? = 0
configure:3287: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3290: $? = 0
configure:3297: test -s conftest.o
configure:3300: $? = 0
configure:3396: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c  >&5
configure:3399: $? = 0
configure:3405: ./conftest
configure:3408: $? = 0
configure:3425: result: yes
configure:3449: checking for sys/types.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for sys/stat.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for stdlib.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for string.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for memory.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for strings.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for inttypes.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for stdint.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3449: checking for unistd.h
configure:3470: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3476: $? = 0
configure:3483: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3486: $? = 0
configure:3493: test -s conftest.o
configure:3496: $? = 0
configure:3509: result: yes
configure:3531: checking math.h usability
configure:3548: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:3554: $? = 0
configure:3561: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3564: $? = 0
configure:3571: test -s conftest.o
configure:3574: $? = 0
configure:3585: result: yes
configure:3589: checking math.h presence
configure:3604: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
configure:3610: $? = 0
configure:3631: result: yes
configure:3659: checking for math.h
configure:3666: result: yes
configure:3680: checking for main in -lm
configure:3709: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c -lm   >&5
configure:3715: $? = 0
configure:3722: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3725: $? = 0
configure:3732: test -s conftest
configure:3735: $? = 0
configure:3749: result: yes
configure:3770: checking for tgamma
configure:3826: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c -lm  >&5
conftest.c:42:6: warning: incompatible redeclaration of library function 'tgamma' [-Wincompatible-library-redeclaration]
char tgamma ();
     ^
conftest.c:42:6: note: 'tgamma' is a builtin with type 'double (double)'
1 warning generated.
configure:3832: $? = 0
configure:3839: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:3842: $? = 0
configure:3849: test -s conftest
configure:3852: $? = 0
configure:3866: result: yes
configure:4176: checking complex.h usability
configure:4193: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:4199: $? = 0
configure:4206: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:4209: $? = 0
configure:4216: test -s conftest.o
configure:4219: $? = 0
configure:4230: result: yes
configure:4234: checking complex.h presence
configure:4249: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
configure:4255: $? = 0
configure:4276: result: yes
configure:4304: checking for complex.h
configure:4311: result: yes
configure:4326: checking for clog
configure:4382: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c -lm  >&5
conftest.c:43:6: warning: incompatible redeclaration of library function 'clog' [-Wincompatible-library-redeclaration]
char clog ();
     ^
conftest.c:43:6: note: 'clog' is a builtin with type '_Complex double (_Complex double)'
1 warning generated.
configure:4388: $? = 0
configure:4395: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:4398: $? = 0
configure:4405: test -s conftest
configure:4408: $? = 0
configure:4422: result: yes
configure:4438: checking for cexp
configure:4494: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c -lm  >&5
conftest.c:44:6: warning: incompatible redeclaration of library function 'cexp' [-Wincompatible-library-redeclaration]
char cexp ();
     ^
conftest.c:44:6: note: 'cexp' is a builtin with type '_Complex double (_Complex double)'
1 warning generated.
configure:4500: $? = 0
configure:4507: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:4510: $? = 0
configure:4517: test -s conftest
configure:4520: $? = 0
configure:4534: result: yes
configure:4569: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
configure:4575: $? = 0
configure:4582: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:4585: $? = 0
configure:4592: test -s conftest.o
configure:4595: $? = 0
configure:4706: checking gridnodes.h usability
configure:4723: gcc -c -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include conftest.c >&5
In file included from conftest.c:56:
/Users/kokim/anaconda3/envs/py37/include/gridnodes.h:15:2: warning: '_GRIDNODES_H' is used as a header guard here, followed by #define of a different macro [-Wheader-guard]
#if !defined(_GRIDNODES_H)
 ^~
/Users/kokim/anaconda3/envs/py37/include/gridnodes.h:16:9: note: 'GRIDNODES_H' is defined here; did you mean '_GRIDNODES_H'?
#define GRIDNODES_H
        ^~~~~~~~~~~
        _GRIDNODES_H
1 warning generated.
configure:4729: $? = 0
configure:4736: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:4739: $? = 0
configure:4746: test -s conftest.o
configure:4749: $? = 0
configure:4760: result: yes
configure:4764: checking gridnodes.h presence
configure:4779: gcc -E -I/Users/kokim/anaconda3/envs/py37/include conftest.c
In file included from conftest.c:23:
/Users/kokim/anaconda3/envs/py37/include/gridnodes.h:15:2: warning: '_GRIDNODES_H' is used as a header guard here, followed by #define of a different macro [-Wheader-guard]
#if !defined(_GRIDNODES_H)
 ^~
/Users/kokim/anaconda3/envs/py37/include/gridnodes.h:16:9: note: 'GRIDNODES_H' is defined here; did you mean '_GRIDNODES_H'?
#define GRIDNODES_H
        ^~~~~~~~~~~
        _GRIDNODES_H
1 warning generated.
configure:4785: $? = 0
configure:4806: result: yes
configure:4834: checking for gridnodes.h
configure:4841: result: yes
configure:4863: checking for gridnodes_readnextpoint in -lgu
configure:4898: gcc -o conftest -I/Users/kokim/anaconda3/envs/py37/include -I/Users/kokim/anaconda3/envs/py37/include -L/Users/kokim/anaconda3/envs/py37/lib conftest.c -lgu  -lm  >&5
configure:4904: $? = 0
configure:4911: test -z "$ac_c_werror_flag" || test ! -s conftest.err
configure:4914: $? = 0
configure:4921: test -s conftest
configure:4924: $? = 0
configure:4938: result: yes
configure:5063: creating ./config.status

## ---------------------- ##
## Running config.status. ##
## ---------------------- ##

This file was extended by config.status, which was
generated by GNU Autoconf 2.60.  Invocation command line was

  CONFIG_FILES    = 
  CONFIG_HEADERS  = 
  CONFIG_LINKS    = 
  CONFIG_COMMANDS = 
  $ ./config.status 

on kokimui-MacBookPro.local

config.status:586: creating makefile
config.status:586: creating config.h
config.status:811: config.h is unchanged

## ---------------- ##
## Cache variables. ##
## ---------------- ##

ac_cv_c_compiler_gnu=yes
ac_cv_env_CC_set=
ac_cv_env_CC_value=
ac_cv_env_CFLAGS_set=set
ac_cv_env_CFLAGS_value=-I/Users/kokim/anaconda3/envs/py37/include
ac_cv_env_CPPFLAGS_set=set
ac_cv_env_CPPFLAGS_value=-I/Users/kokim/anaconda3/envs/py37/include
ac_cv_env_CPP_set=
ac_cv_env_CPP_value=
ac_cv_env_LDFLAGS_set=set
ac_cv_env_LDFLAGS_value=-L/Users/kokim/anaconda3/envs/py37/lib
ac_cv_env_build_alias_set=
ac_cv_env_build_alias_value=
ac_cv_env_host_alias_set=
ac_cv_env_host_alias_value=
ac_cv_env_target_alias_set=
ac_cv_env_target_alias_value=
ac_cv_func_cexp=yes
ac_cv_func_clog=yes
ac_cv_func_tgamma=yes
ac_cv_header_complex_h=yes
ac_cv_header_gridnodes_h=yes
ac_cv_header_inttypes_h=yes
ac_cv_header_math_h=yes
ac_cv_header_memory_h=yes
ac_cv_header_stdc=yes
ac_cv_header_stdint_h=yes
ac_cv_header_stdlib_h=yes
ac_cv_header_string_h=yes
ac_cv_header_strings_h=yes
ac_cv_header_sys_stat_h=yes
ac_cv_header_sys_types_h=yes
ac_cv_header_unistd_h=yes
ac_cv_lib_gu_gridnodes_readnextpoint=yes
ac_cv_lib_m=ac_cv_lib_m_main
ac_cv_lib_m_main=yes
ac_cv_objext=o
ac_cv_path_EGREP='/usr/bin/grep -E'
ac_cv_path_GREP=/usr/bin/grep
ac_cv_path_install='/usr/bin/install -c'
ac_cv_prog_AR=ar
ac_cv_prog_CPP='gcc -E'
ac_cv_prog_ac_ct_CC=gcc
ac_cv_prog_cc_c89=
ac_cv_prog_cc_g=yes

## ----------------- ##
## Output variables. ##
## ----------------- ##

AR='ar'
CC='gcc'
CFLAGS='-I/Users/kokim/anaconda3/envs/py37/include'
CFLAGS_TRIANGLE='-I/Users/kokim/anaconda3/envs/py37/include -w -ffloat-store'
CPP='gcc -E'
CPPFLAGS='-I/Users/kokim/anaconda3/envs/py37/include'
DEFS='-DHAVE_CONFIG_H'
ECHO_C='ECHO_N=''
ECHO_T=''
EGREP='/usr/bin/grep -E'
EXEEXT=''
GREP='/usr/bin/grep'
HAVE_GRIDNODES_H=''
INSTALL_DATA='${INSTALL} -m 644'
INSTALL_PROGRAM='${INSTALL}'
INSTALL_SCRIPT='${INSTALL}'
LDFLAGS='-L/Users/kokim/anaconda3/envs/py37/lib'
LIBOBJS=''
LIBS='-lgu -lm '
LTLIBOBJS=''
NOCOMPLEX=''
OBJEXT='o'
PACKAGE_BUGREPORT=''
PACKAGE_NAME=''
PACKAGE_STRING=''
PACKAGE_TARNAME=''
PACKAGE_VERSION=''
PATH_SEPARATOR=':'
SHELL='/bin/sh'
ac_ct_CC='gcc'
bindir='${exec_prefix}/bin'
build_alias=''
datadir='${datarootdir}'
datarootdir='${prefix}/share'
docdir='${datarootdir}/doc/${PACKAGE}'
dvidir='${docdir}'
exec_prefix='${prefix}'
host_alias=''
htmldir='${docdir}'
includedir='${prefix}/include'
infodir='${datarootdir}/info'
libdir='/Users/kokim/anaconda3/envs/py37/lib'
libexecdir='${exec_prefix}/libexec'
localedir='${datarootdir}/locale'
localstatedir='${prefix}/var'
mandir='${datarootdir}/man'
oldincludedir='/usr/include'
pdfdir='${docdir}'
prefix='/Users/kokim/anaconda3/envs/py37'
program_transform_name='s,x,x,'
psdir='${docdir}'
sbindir='${exec_prefix}/sbin'
sharedstatedir='${prefix}/com'
sysconfdir='${prefix}/etc'
target_alias=''

## ----------- ##
## confdefs.h. ##
## ----------- ##

#define PACKAGE_NAME ""
#define PACKAGE_TARNAME ""
#define PACKAGE_VERSION ""
#define PACKAGE_STRING ""
#define PACKAGE_BUGREPORT ""
#define STDC_HEADERS 1
#define HAVE_SYS_TYPES_H 1
#define HAVE_SYS_STAT_H 1
#define HAVE_STDLIB_H 1
#define HAVE_STRING_H 1
#define HAVE_MEMORY_H 1
#define HAVE_STRINGS_H 1
#define HAVE_INTTYPES_H 1
#define HAVE_STDINT_H 1
#define HAVE_UNISTD_H 1
#define HAVE_LIBM 1
#define HAVE_TGAMMA 1
#define HAVE_CLOG 1
#define HAVE_CEXP 1
#define HAVE_COMPLEX 1
#define HAVE_GRIDNODES_H 1
#define HAVE_LIBGU 1

configure: exit 0
